### ISSUE TITLE
Implicit use of unlabelled information in Transductive fine-tuning

### ISSUE THREAD

**Issue by johncai117** [IBM/cdfsl-benchmark#22]
Hi,

I was just wondering if there was an implicit use of unlabelled data in the process of transductive fine-tuning in the baseline paper. This is because transductive fine-tuning uses the batch norm statistics to fine-tune, and thus we are not predicting each query image in the batch independently of other query images in the batch. 

This is an issue, because based on the above explanation, transduction fine-tuning should only be allowed for Track 2 and not for Track 1 results? An alternative is for the transductive fine-tuning to only use the statistics of the support images - then this should be permitted in Track 1. May I kindly ask which version this paper has implemented?

**Comment by yunhuiguo** [comment]
Hi @johncai117 , the paper has implemented the first version you talked about. You are right, for this particular transductive learning using batch norm statistics the query images are not predicted independently. Other transductive learning methods are also possible, so for Track 2 and Track 1 results,  it depends on the exact method used.
